src/Identity.Clients/Services/Mappers/ResourceMapper.cs

```diff
@@ -15,17 +15,26 @@ public ResourceMapper()
 
             CreateMap<Data.Resource, Resource>()
                 .ForMember(d => d.DisplayName, opt => opt.MapFrom(s => string.IsNullOrEmpty(s.DisplayName) ? s.Name : s.DisplayName))
-                .ForMember(d => d.Description, opt => opt.MapFrom(s => System.String.Join(", ", s.Claims.Select(c => c.Type))));
+                .ForMember(d => d.Description, opt => opt.MapFrom(s => s.UserClaims ?? s.Scopes));
+            CreateMap<Data.Resource, ResourceDetail>()
+                .ForMember(d => d.DisplayName, opt => opt.MapFrom(s => string.IsNullOrEmpty(s.DisplayName) ? s.Name : s.DisplayName))
+                .ForMember(d => d.Description, opt => opt.MapFrom(s => s.UserClaims ?? s.Scopes));
```

Comment:
This description used to be mapped from joining all the claims of a resource. We now have a `UserClaims` field and a `Scopes` field, so perhaps we don't need to map the description to anything anymore.

The only places I've seen the resource description be used is in the Consent `Index.cshtml` page showing a table of Identity Resources with Scope and Claims and in Indentity-UI `api-browser-component.html` also showing a table with scope and claims. 

Can we just remove this mapping and use the correct fields  in these cases? If not, what should the description be mapped to?